editor/src/components/editor/store/editor-state.ts

```diff
@@ -535,27 +873,153 @@ export interface EditorState {
   forceParseFiles: Array<string>
 }
 
+export function editorState(
```

Comment:
mercy the soul who will first receive an error message saying "Expected 61 arguments, but got 62"